lib/mshoplib/src/MShop/Order/Manager/Base/Base.php

```diff
@@ -471,12 +483,19 @@ protected function loadItems( $id, \Aimeos\MShop\Price\Item\Iface $price,
 	protected function loadFresh( $id, \Aimeos\MShop\Price\Item\Iface $price,
 		\Aimeos\MShop\Locale\Item\Iface $localeItem, $row, $parts )
 	{
-		$products = $addresses = $services = [];
+		$products = $coupons = $addresses = $services = [];
 
-		if( $parts & \Aimeos\MShop\Order\Item\Base\Base::PARTS_PRODUCT ) {
+		if( $parts & \Aimeos\MShop\Order\Item\Base\Base::PARTS_PRODUCT
+			|| $parts & \Aimeos\MShop\Order\Item\Base\Base::PARTS_COUPON
```

Comment:
This is wrong because not all products should be loaded if PARTS_COUPON is passed.

Reply:
Right, in the current solution this is not needed. I removed the second condition.